app/src/main/kotlin/com/saltedge/authenticator/features/actions/SubmitActionViewModel.kt

```diff
@@ -61,7 +61,7 @@ class SubmitActionViewModel(
     val onCloseEvent = MutableLiveData<ViewModelEvent<Unit>>()
     val onShowErrorEvent = MutableLiveData<ViewModelEvent<String>>()
     val onOpenLinkEvent = MutableLiveData<ViewModelEvent<Uri>>()
-    val showConnectionsSelectorFragmentEvent = MutableLiveData<List<ConnectionItemViewModel>>()
+    val showConnectionsSelectorFragmentEvent = MutableLiveData<Bundle>()
```

Comment:
now is not event, add ViewModelEvent

Reply:
now is not event - ok
add ViewModelEvent - why? I need just Bundle here

Reply:
purpose of `ViewModelEvent` - one time events like show fragment/dialog. If you think what `ViewModelEvent` is redundant here then remove it from all events designated to show fragment/dialog